frontend/src/components/VExternalSearch/VExternalSearchForm.vue

```diff
@@ -117,7 +117,7 @@ export default defineComponent({
     },
     hasNoResults: {
       type: Boolean,
-      required: true,
+      default: true,
```

Comment:
This prop will be removed in #2811, so I set a default value for it to avoid computing it.